src/Agent.Worker/Worker.cs

```diff
@@ -68,12 +68,6 @@ public async Task<int> RunAsync(string pipeIn, string pipeOut)
                 InitializeSecretMasker(jobMessage);
                 SetCulture(jobMessage);
 
-                if(AgentKnobs.EnableNewSecretMasker.GetValue(HostContext).AsBoolean())
-                {
-                    Trace.Verbose($"{Constants.Variables.Agent.EnableAdditionalMaskingRegexes} is On, adding additional masking regexes");
-                    HostContext.AddAdditionalMaskingRegexes();
```

Comment:
Looks like we used to do this only if "new secret masker" was enabled, but now we will do it for VSO secret masker as well in HostContext. Intended?

Reply:
This is a big problem. See notes.